cloudflare/resource_cloudflare_logpush_ownership_challenge_test.go

```diff
@@ -0,0 +1,40 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCloudflareLogpushOwnershipChallenge(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := "cloudflare_logpush_ownership_challenge." + rnd
+	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
+
+	destinationConf := `s3://jacob-tf-provider-testing-cf-logpush?region=us-east-1`
```

Comment:
@patryk Do you have a S3 or GCP bucket you'd prefer here? This bucket has a lifecycle policy on it to ensure automatic cleanup but up to you.

Reply:
We can use/create GCP bucket we use for acceptance tests.

Reply:
Thanks, totally forgot about this but updated in 5831f35c.